cfgov/jinja2/v1/wagtail/molecules/related-posts-half-width.html

```diff
@@ -0,0 +1,54 @@
+{# ==========================================================================
+
+   related-posts-half-width
+
+   ==========================================================================
+
+   Description:
+
+   Creates related posts markup in half width:
+
+   value:                  An object used to customize the markup.
+
+   value.related_posts:    An array of related posts.
+
+   value.view_more:        An link used to view more related posts.
+
+   ========================================================================== #}
+
+{% import 'macros/time.html' as time %}
+
+<div class="m-related-posts
+            m-related-posts__half-width">
+    <ul class="m-related-posts_list
+               list list__unstyled
+               list__spaced">
+        {% for post in value.related_posts %}
+        <li class="list_item">
+            <h4 class="u-mb5">
```

Comment:
We'll want to change this h4 to an h3 with a class of h4 to maintain heading hierarchy.